addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -118,6 +118,9 @@ function hasColor(element, mode) {
     const style = element.style;
     const parent = element.parentNode;
     const classRegex = mode === 'color' ? TEXT_CLASSES_REGEX : BG_CLASSES_REGEX;
+    if (getComputedStyle(element)[mode] === getComputedStyle(parent)[mode]) {
+        return true;
+    }
```

Comment:
This is already tested below, albeit with additional conditions. Which of the conditions below are not met in your case ?

Reply:
In the case when the mouse pointer is pointing to the default selected color in the color palette, then the colorpicker use to flicker. Anyways I checked it on the runbot now and it seems to be fixed because I am not able to reproduce it.
So, I am removing this fix in the new commit.